### DISCUSSION TITLE
How to use kdbx:// in URL in combination with keyfile + password

### DISCUSSION THREAD

**Discussion by theincogtion** [keepassxreboot/keepassxc#9248] [Q&A] [answered]
I am using kdbx:// in the URL field to open up another database. this works fine as long as there is no keyfile involved. How can I include the path of the keyfile?
I only find options to call keepassxc application itself and use command line arguments. The kdbx:// option is more elegant and it would be great if I just could enter the keyfile path in the advanced fields.
Or do I miss something and this already works?

**Comment by droidmonkey** [comment, marked as answer]
You put it in the username field

**Reply by theincogtion** [reply]
Thanks this works, I couldn't find it in the documentation. Could this be added?
edit:
https://keepassxc.org/docs/KeePassXC_UserGuide.html#_automatic_database_opening
described here :)
